plugins/GSdx/Renderers/DX11/GSDevice11.h

```diff
@@ -309,14 +318,6 @@ class GSDevice11 final : public GSDevice
 		}
```

Comment:
You can purge IsCLR1() too.

Reply:
I think I got it but I did is on the last commit since that was the one where I put in the check for the BLEND_C_CLR flag instead of using the IsCLR1().